pkgs/development/python-modules/vosk/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, srt
+, requests
+, tqdm
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "vosk";
+  version = "0.3.45";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    dist = "py3";
+    platform = "manylinux_2_12_x86_64.manylinux2010_x86_64";
+
+    hash = "sha256-JeAlCTxDmdcnj1Q1aO2MxUYKw6S/SMI2c6zh4l0mYZ8=";
+  };
+
+  propagatedBuildInputs = [
+    srt
+    requests
+    tqdm
+    websockets
+  ];
+
+  meta = with lib; {
```

Comment:
`meta.maintainers` must be set, you can add yourself to maintainer list: 
https://github.com/NixOS/nixpkgs/blob/9ac814f96ac41463c3d0507bc0d8c54d43564303/maintainers/README.md